junit-jupiter-params/src/test/java/org/junit/jupiter/params/ParameterizedTestIntegrationTests.java

```diff
@@ -243,6 +243,14 @@ void executesWithCsvSource() {
 				.haveExactly(1, event(test(), displayName("[2] argument=bar"), finishedWithFailure(message("bar"))));
 	}
 
+	@Test
+	void executesWithRepeatableCsvSource() {
+		var results = execute("testWithRepeatableCsvSource", String.class);
+		results.allEvents().assertThatEvents().haveExactly(1,
+			event(test(), displayName("[1] argument=a"), finishedWithFailure(message("a")))).haveExactly(1,
+				event(test(), displayName("[2] argument=b"), finishedWithFailure(message("b"))));
```

Comment:
```suggestion
		results.allEvents().assertThatEvents() //
				.haveExactly(1, event(test(), displayName("[1] argument=a"), finishedWithFailure(message("a")))) //
				.haveExactly(1, event(test(), displayName("[2] argument=b"), finishedWithFailure(message("b"))));
```

Reply:
Reformatted and made use of `//` in f1046d205ceab97f58aa656818dd0fa6099d7277